### ISSUE TITLE
Subcraft Rotate does not work

### ISSUE THREAD

**Issue by eirikh1996** [AJCStriker/Movecraft-3#143]
I have movecraft installed on my server, and i should try out the subcraft sign to make a turret.

What i expected to happen:
A small wheel should appear on the two lowest lines and the turret should turn.

What really happened:
Nothing happened (the small wheel didnt generate and the subcraft didnt work)

**Comment by mwkaicz** [comment]
It looks like Movecraft can't recognize the sign.
I think that you are just writing something wrong on the sign.
1st: ```Subcraft Rotate```
2nd: ```type``` (use own craft type, f.e: "turret")
3rd and 4th is empty

**Comment by eirikh1996** [comment]
I did as instructed, however, it still does not work
Also, when I upgraded Movecraft, i did not delete the old Movecraft folder or its content
You can see how i did it and that it does not work
![2015-03-25_17 05 11](https://cloud.githubusercontent.com/assets/11066794/6829054/6663d05e-d311-11e4-9f1f-d9b3aa6c72c6.png)
![2015-03-25_17 05 10](https://cloud.githubusercontent.com/assets/11066794/6829055/66677ae2-d311-11e4-8748-8b7ee7be9a7d.png)
![2015-03-25_17 05 09](https://cloud.githubusercontent.com/assets/11066794/6829056/66685250-d311-11e4-88d9-9ccd6d4a2351.png)
![2015-03-25_17 04 56](https://cloud.githubusercontent.com/assets/11066794/6829057/666a8b92-d311-11e4-9abd-e63dffb49ac3.png)

**Comment by mwkaicz** [comment]
Are u sure that one of your files contains name=Turret? If yes and you have latest movecraft this must working. If still doesn't source of this problem must be in another plugin.

**Comment by eirikh1996** [comment]
yes. I have name: Turret. I have the latest Movecraft plugin, however, it still does not work

**Comment by mwkaicz** [comment]
Some error in console?

**Comment by eirikh1996** [comment]
There were no errors in the console, but still, it does not work

**Comment by mwkaicz** [comment]
I don't see any reason why it shouldn't work. Do you have another plugins which using Signs? Try to remove them one by one (just to be sure that this problem doesn't comes from another source)

**Comment by eirikh1996** [comment]
I had CommandSigns and TutorialSigns

**Comment by eirikh1996** [comment]
i uninstalled ALL other plugins that use signs, but it has NO effect at all. Problem is still the same

**Comment by pistols12** [comment]
Make sure your Subcraft and your craft are not sharing the same Block ID.... for example, you have that white wool with the subcraft sign, if that is connected to your ship, it will not rotate if the ship's blocks are also able to be made of White wool.  

In short, you're subcraft, can have no blocks touching the main craft that they both share in terms of ID list. If you had a door, and a door frame for example, the frame for example sake is Oak wood, and Door Dark Oak Logs , with this setup your subcraft (or door) should not be allowed to attatch to OAK, and the Frame, not allowed to have Dark Oak Logs in the craft.  here is a good video describing it all:

https://www.youtube.com/watch?v=YbOezZS06oU

**Comment by otsmania** [comment]
Make sure your "Turret" type works properly
